lexers/go_test.go

```diff
@@ -120,3 +120,29 @@ func TestGoHTMLTemplateNegativeNumber(t *testing.T) {
 		assert.True(t, found)
 	}
 }
+
```

Comment:
Please remove this test, instead add new files to `lexers/testdata` that exercise this change.

Reply:
Oh, excellent. Done.